webtau-core/src/main/java/com/twosigma/webtau/data/table/header/Header.java

```diff
@@ -111,6 +111,16 @@ public String toString() {
         return namesByIndex.toString();
     }
 
+    /**
+     * column index by column name
+     * @param columnName column name to get index for
+     * @return column index or -1 if not found
+     */
+    public int findColumnIdxByName(String columnName) {
+        Integer idx = indexByName.get(columnName);
+        return idx == null ? -1: idx;
```

Comment:
do we want to make `-1` a constant "COLUMN_NOT_FOUND" or some such?

Reply:
Not to keen on that. I think by now -1 is a common knowledge.